.gitignore

```diff
@@ -1,28 +1 @@
-src/_main_/config/*json
```

Comment:
removing this because we have a duplicate .gitignore in the src/ folder that handles most of this